fix: handle situation where gitlab.com does not return values (#1773)
python-gitlab/python-gitlab: docs/api-usage.rst

* ``prev_page``: if ``None`` the current page is the first one
* ``next_page``: if ``None`` the current page is the last one
* ``per_page``: number of items per page
* ``total_pages``: total number of pages available. This may be a ``None`` value.
* ``total``: total number of items in the list. This may be a ``None`` value.

.. note::

>    ``total_pages`` and ``total`` may have a value of ``None``. For performance
>    reasons, if a query returns more than 10,000 records, GitLab doesn’t return
>    any value for ``total_pages`` or ``total``.
> 
>    For more information see:
>    https://docs.gitlab.com/ee/user/gitlab_com/index.html#pagination-response-headers

Reviewer: I think not just the values but the headers are missing entirely, right?

```suggestion
   For performance reasons, if a query returns more than 10,000 records, GitLab
   does not return the ``total_pages`` or ``total`` headers.  In this case,
   ``total_pages`` and ``total`` will have a value of ``None``.

   For more information see:
   https://docs.gitlab.com/ee/user/gitlab_com/index.html#pagination-response-headers
```

Author: Done
